src/pages/Contact.css

```diff
@@ -0,0 +1,138 @@
+.titleContainer{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+
+}
+.title{
+    /* style the CONTACT US */
+    width: 468px;
+    height: 68px;
+    /* margin-left: 468px; */
+    margin-left: 242px;
+    top: 159px;
+    
+
+    font: PingFang SC;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 48px;
+    /* text-align: center;  */
+}
+
+.bottomContainer{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+
+}
+.messageContainer{
+    display: flex;
+    flex-direction: column;
+
+    /* why isn't this working? */
+    align-content: space-between;
+
+    justify-content: center;
+}
+.messageTitle{
+
+     width: 495px;
+     height: 42px;
+     margin-left: 162px; 
+
+
+     text-align: center;
+
+     font-style: normal;
+     font-weight: 600;
+     font-size: 30px;
+}
+.smallInput{
+    /* style the inputs */
+    margin-left: 116px;
+    height: 52px;
+    width: 608px;
```

Comment:
As noted above; let flexbox handle it

Reply:
If I get rid of these dimensions though, the input objects shrink. 
<img width="1344" alt="Screen Shot 2020-11-18 at 8 13 55 PM" src="https://user-images.githubusercontent.com/71359476/99612140-a1420580-29da-11eb-93d3-58aefb22a9fd.png">